spec/app_spec.rb

```diff
@@ -26,33 +26,20 @@ def app
     expect(last_response.body).to include("_ _")
   end
 
-  it "asks to make guess on the homepage" do
-    get '/'
-
-    expect(last_response.body).to include("Make a guess")
-  end
-
-  it "displays list of letters on the homepage" do
-    get '/'
-
-    expect(last_response.body).to include("a b c d e f g h i j k l m n o p q r s t u v w x y z")
-  end
-
-  it "displays the heart image corresponding to start of game on homepage" do
-    starting_image = "heart"
+  it "displays the start image corresponding to start of game on homepage" do
+    starting_image = "start"
 
     get '/'
 
     expect(last_response.body).to include(starting_image)
   end
 
-  it "deletes guessed letter from guessable letters in play route" do
-    get '/'
-    post '/play', 'letter_chosen' => 'a'
+  it "shows wrong guesses" do
+    play("hi", ["x", "o"])
```

Comment:
I see you're still in TTT mode with guessing x and o 😂

Reply:
🤣